### DISCUSSION TITLE
Only show hamburger and not entire menu until required

### DISCUSSION THREAD

**Discussion by wintercorn** [templaza/astroid-framework#750] [Q&A]
I need to show only the hamburger, and not the entire sticky header or menu, until it's clicked on. Apart from using up my entire stock of 'display:none' is there a setting to achieve this?

Thanks

**Comment by sonvnn** [comment]
Unfortunately, there is no such function at the moment. However, you can use custom js and css in the template to create the effects as you wish.

**Comment by wintercorn** [comment]
OK no problems. Thanks for the quick reply!
